### ISSUE TITLE
JS Error on the wallet page (empty wallet)

### ISSUE THREAD

**Issue by sergejmueller** [DeFiCh/app#369]
### JS Error
```
A non-serializable value was detected in an action, in the path: `payload.cancelToken`.
```

### Preflight Checklist
- [x] I have searched [DeFiChain’s Github Issue Page](https://github.com/DeFiCh/app/issues) and none matches my issue
- [x] I have read through [How To Submit Issues for DeFi App](https://github.com/DeFiCh/app/wiki/How-to-submit-issues-for-DeFi-app”)

### Issue Details

- **Current DeFi App Version:**
  - 2.1.12
- **Previous DeFi App Version:**
  - 2.1.11
- **Operating System (OS):**
  - MacOS

### Screenshot(s) / screen recording(s) of issue
Video: https://siasky.net/_ATxUkS_PywvQ2XhBcHb1vQwPM9dTp7kSTbnLY0BvNRo7g
Screenshot: https://siasky.net/fAAZk55CuoyQqVP-kG6Skhvt_Gxlu_J-GVWED6IHIwcLnQ

### Actual Behavior
```
1. Open the app
2. Open Developer Tools
3. Go to wallets page
4. Click on a wallet
5. See the console
```

### Expected Behavior
No JS error occurs

### Logs
[Archiv.zip](https://github.com/DeFiCh/app/files/5860803/Archiv.zip)

**Comment by izzycsy** [comment]
Hi @sergejmueller  	, update [v2.1.15](https://defichain.com/downloads/) is out.

Do give a thumbs up to this comment if your issue has been resolved, otherwise, provide new information (include screenshots) and app logs. Thank you.

**Comment by sergejmueller** [comment]
@izzycsy Not fixed, just the same error.

<img width="1035" alt="Bildschirmfoto 2021-01-31 um 19 21 30" src="https://user-images.githubusercontent.com/108992/106393845-90116e00-63f9-11eb-87ba-1fbf8442b047.png">

**Comment by sergejmueller** [comment]
@izzycsy Wich PR should solve this issue?

**Comment by izzycsy** [comment]
Hi @sergejmueller , this issue is not yet available in the current v2.1.15. Rest assured that your feedback has been sent to the team. I'll keep you updated when it's been fixed. Thank you for your patience.

**Comment by thedoublejay** [comment]
This should be fixed on v2.1.17.
